webapp/src/components/root/root.jsx

```diff
@@ -14,6 +14,7 @@ export default class Root extends React.Component {
         visible: PropTypes.bool.isRequired,
         message: PropTypes.string.isRequired,
         postID: PropTypes.string.isRequired,
+        selectedPost: PropTypes.object.isRequired,
```

Comment:
Instead of selected post would be selected issue, right? Since what it is actually selected is a todo issue.